packages/expo-secure-store/ios/SecureStoreExceptions.swift

```diff
@@ -55,7 +55,7 @@ internal class KeyChainException: GenericException<OSStatus> {
       return "Authentication failed. Provided passphrase/PIN is incorrect or there is no user authentication method configured for this device."
 
     default:
-      return "Unknown Keychain Error."
+      return "Unknown Keychain Error: \(String(describing: SecCopyErrorMessageString(param, nil)))"
```

Comment:
```suggestion
      if let errorMessage = SecCopyErrorMessageString(param, nil) as? String {
        return errorMessage
      }
      return "Unknown Keychain Error."
```

- `String(describing:)` seems not necessary as CFString can convert to String natively
- better handling if `SecCopyErrorMessageString()` returns nil
- if `SecCopyErrorMessageString()` returns valid string, we don't have to include "Unknown" in the message.

Reply:
Thanks, I'm relatively new to Swift so appreciate the nitpicks 😀